SingularityUI/app/actions/api/base.es6

```diff
@@ -2,7 +2,7 @@ import fetch from 'isomorphic-fetch';
 
 const JSON_HEADERS = {'Content-Type': 'application/json', 'Accept': 'application/json'};
 
-export function buildJsonApiAction(actionName, httpMethod, opts={}) {
+export function buildJsonApiAction(actionName, httpMethod, opts={}, keyField=undefined) {
```

Comment:
There's no functional reason, I decided to have that to make it more obvious to the reader that the parameter is explicitly optional.